compat/vcbuild/vcpkg_install.bat

```diff
@@ -53,9 +59,19 @@ REM ================================================================
 	IF ERRORLEVEL 1 ( EXIT /B 1 )
 
 	echo Successfully installed %cwd%vcpkg\vcpkg.exe
+	GOTO :install_libraries
+
+:check_for_updates
+	echo Checking for vcpkg updates and upgrades to installed libraries
+	cd %cwd%vcpkg
+	git pull
+	.\vcpkg.exe update
+	.\vcpkg.exe upgrade --no-dry-run
+
+	EXIT /B 0
 
 :install_libraries
-	SET arch=x64-windows
+REM	SET arch=x64-windows
```

Comment:
Ah, that's where this comes from. Urgh.

So you are just moving the definition. But you're not, you're commenting out the old code, making it "dead code". Please don't.

And if you are moving the definition, please move it to the top, with a comment that the architecture is currently hard-coded, and that we will change that once we need to.

Reply:
"Dead code" - I'll keep it (typically) until we have working code (the point of requesting the support/review). 

> the architecture is currently hard-coded, and that we will change that once we need to

I can add your comment.